docker/zap-api-scan.py

```diff
@@ -206,8 +208,13 @@ def main(argv):
             timeout = int(arg)
         elif opt == '-O':
             host_override = arg
+        elif opt == '--hook':
+            hook_file = arg
 
     check_zap_client_version()
+    
+    load_custom_hooks(hook_file)
+    opts = trigger_hook('cli_opts', opts)
```

Comment:
What is this one for? Or, does it worth assigning the return value even if `opts` is never used after?